From df7ba946b6db35845a77551096c6bb62a00e289f Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 22 Jan 2018 23:14:46 +0800
Subject: [PATCH 3232/5242] MLK-17440 ARM: dts: imx7ulp-evk-qspi: update cpu
 opp table

commit  bd423013b05cf4f4eba497ae4282a68fe3cd8d1e from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX7ULP B0 has different CPU opp table, update it accordingly.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Ye Li <ye.li@nxp.com>
Tested-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk-qspi.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk-qspi.dts b/arch/arm/boot/dts/imx7ulp-evk-qspi.dts
index e760588..67b7c72 100644
--- a/arch/arm/boot/dts/imx7ulp-evk-qspi.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk-qspi.dts
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2016 Freescale Semiconductor, Inc.
- * Copyright 2017 NXP
+ * Copyright 2018 NXP
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 as
@@ -43,7 +43,7 @@
 	arm-supply= <&dummy>;
 	operating-points = <
 		/* KHz  uV */
-		503666  1025000
+		500210	1025000
 	>;
 };
 
-- 
1.7.9.5

